validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -159,6 +159,8 @@ def can_set(self, address_value_list):
         for add_value_dict in address_value_list:
             for address in add_value_dict.keys():
                 if address not in self._write_list:
+                    LOGGER.warning("Authorization exception, address: %s",
```

Comment:
Given that we properly handle this exception, we don't really need to log here. It could just create log noise, that no one can really deal with.